src/controllers/movies.js

```diff
@@ -0,0 +1,63 @@
+const { Prisma } = require('@prisma/client')
+const prisma = require('../utils/prisma')
+
+const getMovieList = async (req, res) => {
+	const movies = await prisma.movie.findMany({
+		orderBy: {
+			id: 'asc',
+		},
+		include: {
+			screenings: true,
+		},
+	})
+	return res.status(200).json({ movies: movies })
+}
+
+const createMovie = async (req, res) => {
+	const body = req.body
+	const newMovie = await prisma.movie.create({
+		data: {
+			title: body.title,
+			runtimeMins: body.runtimeMins,
+		},
```

Comment:
I like how you've handled passing in the body here to use for assigning! Nice and clear where the data is coming from and going to

Reply:
Yeah, I think sometimes is good to write things this way so if, at some point at any time, I will go again through this code, I exactly know what's going on. 
Thank you:)